practices/team_roles_definition.md

```diff
@@ -104,6 +104,7 @@ Belbin recognized that the team success depends on the behavior of the team memb
 
 - Belbin: [The Coming Shape Of Organization](http://www.belbin.ie/the-coming-shape-of-organization/)
 - Medium: [Software Project Team Roles and Responsibilities](https://medium.com/@SherrieRose/software-project-team-roles-and-responsibilities-152a7d575759)
+- Forbytes: [Making sense of the roles in a software project](https://forbytes.com/frequently-asked-questions/roles-and-responsibilities/)
```

Comment:
the link is invalid - https://forbytes.com/frequently-asked-questions/roles-and-responsibilities/

remove it

Reply:
I tried to remove it twice in the past, I hope it's done this time!